pkg/edit/complete/complete_test.go

```diff
@@ -110,7 +110,7 @@ func TestComplete(t *testing.T) {
 	}
 
 	allFileNameItems := []completion.Item{
-		fc("a.exe", " "), fc("d" + string(os.PathSeparator), ""), fc("non-exe", " "),
+		fc("a.exe", " "), fc("d"+string(os.PathSeparator), ""), fc("non-exe", " "),
```

Comment:
This change is the result of `go fmt`. Since it was the only line in the existing code that was reformatted I decided to just include it even though it is not otherwise relevant to this PR.